--- guacamole-server.init	2020-04-24 12:15:52.531203812 +0200
+++ guacamole-server.init	2020-04-09 21:35:54.005093702 +0200
@@ -19,16 +19,17 @@
 # Source function library.
 . /etc/rc.d/init.d/functions
 
+user=guacd
 exec="/usr/sbin/guacd"
 prog="guacd"
-pid="/var/run/$prog.pid"
-OPTS="$OPTS -p $pid"
+pid="/var/run/guacd/$prog.pid"
 
 [ -e /etc/sysconfig/$prog ] && . /etc/sysconfig/$prog
 
+OPTS="$OPTS -p $pid"
+
 # Ensure $HOME is set properly, even if environment is clear
-USER=`whoami`
-export HOME=`sh -c "echo ~$USER"`
+export HOME=`sh -c "echo ~$user"`
 
 lockfile=/var/lock/subsys/$prog
 
@@ -36,7 +37,7 @@
     [ -x $exec ] || exit 5
     [ -f $config ] || exit 6
     echo -n $"Starting $prog: "
-    daemon $prog $OPTS
+    daemon --user "$user" $prog $OPTS
     retval=$?
     echo
     [ $retval -eq 0 ] && touch $lockfile
