--- jmorecfg.h	2017-12-14 04:39:01.000000000 +0000
+++ jmorecfg.h	2020-04-05 20:41:25.974941233 +0000
@@ -121,6 +121,7 @@
  * typedefs live at a different point on the speed/space tradeoff curve.)
  */
 
+#if 0
 /* UINT8 must hold at least the values 0..255. */
 
 #ifdef HAVE_UNSIGNED_CHAR
@@ -140,6 +141,7 @@
 #else /* not HAVE_UNSIGNED_SHORT */
 typedef unsigned int UINT16;
 #endif /* HAVE_UNSIGNED_SHORT */
+#endif
 
 /* INT16 must hold at least the values -32768..32767. */
 
